### PR TITLE
Rtt 10 multiple instructors scheduled cost

### PR DESCRIPTION
**Pull request by eduranperez**

### CODE REVIEW

**Review by flcli — APPROVED** [review]

File: src/main/java/edu/ucdavis/dss/ipa/api/components/budget/views/BudgetComparisonExcelView.java [hunk 1]
```diff
@@ -98,8 +98,8 @@ protected void buildExcelDocument(Map<String, Object> model, Workbook workbook,
             ExcelHelper.writeRowToSheet(report, Arrays.asList(""));
 
             ExcelHelper.writeRowToSheet(report, Arrays
-                .asList("Categories", "Total Cost", "# Courses", "",
-                    "Categories", "Total Cost", "# Courses", "", "Cost", "# Courses", "% Cost",
+                .asList("Categories", "Total Cost", "# Assignments"/*"# Courses"*/, "",
```
Comment (jaroldwong) [inline src/main/java/edu/ucdavis/dss/ipa/api/components/budget/views/BudgetComparisonExcelView.java:101]: should this just say assignments?

Comment (flcli) [reply src/main/java/edu/ucdavis/dss/ipa/api/components/budget/views/BudgetComparisonExcelView.java:101]: I like this idea

Comment (eduranperez) [reply src/main/java/edu/ucdavis/dss/ipa/api/components/budget/views/BudgetComparisonExcelView.java:101]: sure thing I'll switch it.

File: src/main/java/edu/ucdavis/dss/ipa/api/components/budget/views/BudgetExcelView.java [hunk 2]
```diff
@@ -155,7 +163,7 @@ protected void buildExcelDocument(Map<String, Object> model, Workbook workbook,
                                 sectionGroupCost.getEnrollment(),
                                 currentEnrollment,
                                 sectionGroupCost.getSectionCount(),
-                                (sectionGroupCost.getInstructor() == null ? "" : sectionGroupCost.getInstructor().getFullName()),
+                                String.join(", ", instructors),
```
Comment (jaroldwong) [inline src/main/java/edu/ucdavis/dss/ipa/api/components/budget/views/BudgetExcelView.java:166]: It looks like previously (before the multiple budget scenario download work) the Excel download included the instructor type assignments under the Instructor(s) column e.g. "Continuing Lecturer", "Associate Instructor"

I don't know if we want to fix it in this PR or make a new one.

Comment (flcli) [reply src/main/java/edu/ucdavis/dss/ipa/api/components/budget/views/BudgetExcelView.java:166]: This PR is fine. Good catch.

File: src/main/java/edu/ucdavis/dss/ipa/services/jpa/JpaTeachingAssignmentService.java [hunk 3]
```diff
@@ -50,7 +50,7 @@ public TeachingAssignment save(TeachingAssignment teachingAssignment) {
 	@Override
 	@Transactional
 	public void delete(Long id) {
-		teachingAssignmentRepository.deleteById(id);
+		teachingAssignmentRepository.delete(id);
```
Comment (jaroldwong) [inline src/main/java/edu/ucdavis/dss/ipa/services/jpa/JpaTeachingAssignmentService.java:53]: do we still need the `deleteById` method in `TeachingAssignmentRepository.java`?

Comment (flcli) [reply src/main/java/edu/ucdavis/dss/ipa/services/jpa/JpaTeachingAssignmentService.java:53]: Identical to `delete(id)`? Nomenclature issue?

Comment (eduranperez) [reply src/main/java/edu/ucdavis/dss/ipa/services/jpa/JpaTeachingAssignmentService.java:53]: Not identical but no longer used.  Ill delete it.  delete is an out of the box function but deletebyId was custom.

File: src/main/java/edu/ucdavis/dss/ipa/api/components/budget/views/BudgetExcelView.java (hunk 2 above)
Comment (eduranperez) [reply src/main/java/edu/ucdavis/dss/ipa/api/components/budget/views/BudgetExcelView.java:166]: sounds good, I'll add that in.

Comment (jaroldwong) [reply src/main/java/edu/ucdavis/dss/ipa/api/components/budget/views/BudgetExcelView.java:168]: I tried it again and it doesn't look like it's picking up the instructor type descriptions?

Comment (eduranperez) [reply src/main/java/edu/ucdavis/dss/ipa/api/components/budget/views/BudgetExcelView.java:168]: hmm yeah forgot to take into account teaching assignments without instructor costs.  My change only fixed it for non live data or live data if the cost is set explicitly.

**Pull request merged**